.travis.yml

```diff
@@ -163,10 +163,10 @@ matrix:
   #  os: osx
 
   # --------------------------------------------------------------------------
-  # Lint e.g. HLINT_COMMANDS="hlint lint src; hlint lint test"
+  # Lint
   # --------------------------------------------------------------------------
 
-  #- env: BUILD=stack RESOLVER=lts-15 HLINT_COMMANDS="hlint lint ."
+  #- env: BUILD=hlint HLINT_TARGETS="src test"
```

Comment:
We don't know the directory structure of arbitrary users. its better to keep it ".".

Reply:
I'm keeping it `src`.